modules/cactus-web/src/Modal/Modal.tsx

```diff
@@ -78,9 +83,14 @@ export const ModalPopUp = styled(DialogOverlay)<ModalPopupProps>`
     margin: auto;
     max-width: ${(p) => !p.width && '80%'};
     ${width}
+    ${height}
+    ${maxHeight}
     outline: none;
     padding: 64px 24px 40px 24px;
     position: relative;
+    .modal-children {
+      word-break: break-word;
```

Comment:
This is still overflowing in IE11.  I think we need to replace it with the following, as we do elsewhere:

```suggestion
      overflow-wrap: break-word;
      word-wrap: break-word;
```

Reply:
Hmmmm, that's a problem then. I originally wanted to do this with `overflow-wrap`, but I kept running into issues with it being overridden. The weird thing is: The value it kept overwriting it with _was_ `break-word`, so `overflow-wrap: break-word` doesn't work here. I'll have to see if I can figure out a different way, unfortunately.

Reply:
I think the problem had something to do with some conflicting Flex styles...I added a content div fo the children and applied the styles there, which seems to have worked.